libp2p/Host.cpp

```diff
@@ -850,6 +844,19 @@ bytes Host::saveNetwork() const
 
```

Comment:
FYI: This `peers.sort()` is buggy. It works by converting `Peer` objects to bool. Probably not what you wanted. You need to define `operator<` for `Node`.

Reply:
I removed sorting https://github.com/ethereum/aleth/issues/5470#issuecomment-461088152